aggregate/billing_functions/utils.py

```diff
@@ -0,0 +1,742 @@
+# pylint: disable=global-statement,too-many-arguments
+"""
+Class of helper functions for billing aggregate functions
+"""
+import asyncio
+import os
+import re
+import json
+import logging
+from pathlib import Path
+from io import StringIO
+from copy import deepcopy
+from collections import defaultdict
+from datetime import datetime, timedelta
+from typing import Dict, List, Any, Iterator, Optional, Sequence, Tuple, TypeVar, Union
```

Comment:
We could consider using Python 3.9's new typing syntax

Reply:
I've tested these, just need to use square brackets of parenthesis, eg(`dict[str, str]` ~`dict(str, str)`~)